File: extract.js
const fs = require('fs');
const pdfParse = require('pdf-parse');

const files = ['public/Privacy.pdf', 'public/Refund.pdf', 'public/T&C.pdf'];

async function run() {
  for (let file of files) {
    let dataBuffer = fs.readFileSync(file);
    let data = await pdfParse(dataBuffer);
    fs.writeFileSync(file + '.txt', data.text);
    console.log(`Extracted ${file}`);
  }
}
run();

File: extract.py
import PyPDF2
import os

files = ['public/Privacy.pdf', 'public/Refund.pdf', 'public/T&C.pdf']

for file in files:
    if os.path.exists(file):
        reader = PyPDF2.PdfReader(file)
        text = ''
        for page in reader.pages:
            text += page.extract_text() + '\n'
        with open(file + '.txt', 'w', encoding='utf-8') as f:
            f.write(text)
        print(f'Extracted {file}')
    else:
        print(f'File not found: {file}')
